tests/v2/validation/rbac/rbac.go

```diff
@@ -170,3 +181,58 @@ func deleteLabels(labels map[string]string) {
 		}
 	}
 }
+
+func convertSetting(steveAdminClient *v1.Client, globalSetting *v1.SteveAPIObject) (*v3.Setting, error) {
+	updateSetting := &v3.Setting{}
+	err := v1.ConvertToK8sType(globalSetting.JSONResp, updateSetting)
+	if err != nil {
+		return nil, err
+	}
+	return updateSetting, err
+}
+
+func listGlobalSettings(steveclient *v1.Client) (projectNames []string, err error) {
+	globalSettings, err := steveclient.SteveType("management.cattle.io.setting").List(nil)
+	if err != nil {
+		return nil, err
```

Comment:
To expand my above explanation, if we were only using return here, we would send `projectNames` as `nil` slice and err with an assigned err value.

Please reach me offline if you have any questions regarding this.

Reply:
Done. Couldn't resist in changing other functions with named return variables too :)